src/Normalizer/EntityReferenceItemNormalizer.php

```diff
@@ -78,11 +80,13 @@ public function normalize($field_item, $format = NULL, array $context = array())
       // only the predicate? Well because i hope i will be able
       // to MAP to RDF also sub fields of a complex field someday
       // and somehow.
-      $field_mappings = $context['current_entity_rdf_mapping']->getPreparedFieldMapping($field_item->getParent()
-        ->getName());
+      $field_mappings = $context['current_entity_rdf_mapping']->getPreparedFieldMapping(
```

Comment:
@ruebot for sake of easier reading, i feel that maybe $field_item->getParent()->getName() should all go in the same line.